integration/testdata/alpine-distroless.json.golden

```diff
@@ -79,6 +82,40 @@
           "LastModifiedDate": "2022-04-12T21:15:00Z"
         }
       ]
+    },
+    {
+      "Target": "usr/bin/git-lfs",
```

Comment:
Perhaps this is system file and we didn't remove it.

Reply:
Honestly, I don't know how this appeared here but I don't see how it can be related to my changes.

Reply:
I checked this case.
Looks like this problem because `result` is nil:
https://github.com/aquasecurity/trivy/blob/5704a8fda1f86f7f16d18450f7849dd0c1044e28/pkg/fanal/artifact/image/image.go#L245-L247
Handlers don't work

Reply:
I thought about this case and found 1 more way.
I made these changes in another branch (I did not update client/server mode and SBOM) – https://github.com/DmitriyLewen/trivy/commit/20a889290eeae04c2b97e681c9ede385a488401d
I think this way is easier.
@juan131 Can you take a look?

Reply:
applied your suggestion at https://github.com/aquasecurity/trivy/pull/5439/commits/d2222197d294178711b19d065aaebaf976b73b37